src/Registers.h

```diff
@@ -33,6 +33,8 @@ const uintptr_t X86_DF_FLAG = 1 << 10;
 const uintptr_t X86_RF_FLAG = 1 << 16;
 const uintptr_t X86_ID_FLAG = 1 << 21;
 
+const uintptr_t AARCH64_DBG_SPSR_SS = 1 << 21;
```

Comment:
I'm just assuming this is correct because it's shockingly difficult to find what the layout of the PSTATE is in the ARM Reference Manual.

Reply:
> it's shockingly difficult to find what the layout of the PSTATE is in the ARM Reference Manual.

right? And yes, I think it's correct.